packages/components/addon/components/hds/dropdown/header.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="hds-dropdown__header{{if @hasSeparator ' hds-dropdown__header--separator'}}" ...attributes>
```

Comment:
```suggestion
<div class="hds-dropdown__header {{if @hasSeparator 'hds-dropdown__header--separator'}}" ...attributes>
```

Reply:
I tend to do this to have clean mark-up (`"hds-dropdown__header"` instead of ``"hds-dropdown__header "``, but I appreciate it's easier to read in the suggested form so I've updated accordingly